### PR TITLE
closes #12747553 | Politicians Filter

### PR DESCRIPTION
**Pull request by Saint-lawal**
### What does this PR do?
- It adds filter to the politicians page, using:
  -  Name
  -  Political Party
  - State
**Note**: Political Position was not added due to the use of political positions in current tab.

Screen Shot 2019-12-14 at 1 46 34 AM

Screen Shot 2019-12-14 at 2 11 42 AM
Screen Shot 2019-12-14 at 2 32 51 AM
Screen Shot 2019-12-16 at 10 26 24 PM
Screen Shot 2019-12-16 at 10 25 48 PM
Screen Shot 2019-12-14 at 2 15 35 PM

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  value: 'minister',
+}), Object({
+  label: 'Commissioner',
+  value: 'commissioner',
+}), Object({
+  label: 'Local Government Chairman',
+  value: 'local',
+})];
+
+export const states = [
```
Comment (chinezenwosu) [inline src/constants/constants.js]: I feel like we could have this as its own json data file

Comment (Saint-lawal) [reply src/constants/constants.js]: Done @chinezenwosu

File: src/components/Dropdown.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<template>
```
[the remark this answers is not in the document]
Comment (Saint-lawal) [reply src/components/Dropdown.vue]: Done @chinezenwosu

File: src/components/Dropdown/Dropdown.vue [hunk 3]
```diff
@@ -1,10 +1,11 @@
 <template>
-  <button class="relative text-left focus:outline-none" @focusout="closeDropdown">
+  <button class="relative text-left focus:outline-none" :class="parentWidth || 'w-auto'" @focusout="closeDropdown">
```
Comment (chinezenwosu) [inline src/components/Dropdown/Dropdown.vue]: Any reason you made these changes to the dropdown? You've destroyed the mobile view.

Comment (Saint-lawal) [reply src/components/Dropdown/Dropdown.vue]: lol destroyed? the reason I made it is cause it wouldn't fill the parent div, and hence some text were being cut out. How did it "destroy" the mobile view?

Comment (chinezenwosu) [reply src/components/Dropdown/Dropdown.vue]: Huh?

I still need to fix an issue I just noticed with the dropdowns though but not with the way you changed it 😏
**Before vs after**

Before
Screenshot 2019-12-16 at 7 00 44 PM
Screenshot 2019-12-16 at 7 17 10 PM

After
Screenshot 2019-12-16 at 6 59 39 PM
Screenshot 2019-12-16 at 6 59 49 PM

Comment (Saint-lawal) [reply src/components/Dropdown/Dropdown.vue]: Yeah, the reason why this couldn't have worked, is cause it ended up being a top page dropdown for **EVERY** dropdown used on a small screen 😓 @chinezenwosu

**Pull request merged**